src/db/collections/analytics/protocol_parameters.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use async_trait::async_trait;
+use mongodb::error::Error;
+
+use super::{Analytic, Measurement, PerMilestone};
+use crate::{
+    db::{collections::ProtocolUpdateCollection, MongoDb},
+    types::{stardust::milestone::MilestoneTimestamp, tangle::MilestoneIndex},
+};
+
+/// Computes the statistics about the token claiming process.
+#[derive(Debug)]
+pub struct ProtocolParametersAnalytics;
+
+#[async_trait]
+impl Analytic for ProtocolParametersAnalytics {
```

Comment:
Taking this as an example: Why does the trait use the singular, i.e. `Analytic`, but the implementor the the plural `...Analytics`.